Cliges/graph.gv

```diff
@@ -0,0 +1,20 @@
+digraph {
+    O [color="grey"]
+    alpha [label="α", color="grey"]
+    beta [label="β", color="grey"]
+    betaPrime [label="β'", color="grey"]
+    gamma [label="γ", color="grey"]
+
+    O -> alpha;
+    O -> beta;
+    alpha -> AN;
```

Comment:
wondering, here: A and N are two different witnesses ? Should we interpret it s
```
alpha -> 1;
1 -> A;
1 -> N
1[label="", color="grey"]
```
idem for CRT

Reply:
Yes, I think we don't want to transcribe the image printed, but to translate the stemma that was meant to be expressed by that image; if this makes sense.

Reply:
Completely agree, and we should start adding such choices to Guidelines as Simon said. I've created an issue for that in #32